justfile

```diff
@@ -0,0 +1,9 @@
+_default:
+  @just --list
+
+clean:
+  rm -r themes | true
```

Comment:
> The | true on the rm command makes sure that the justfile doesn't error if the themes directory didn't already exist.

`rm -rf` doesn't output a non zero exit code if it tries to remove a directory that doesn't exist, see my screenshot below:

![image](https://github.com/user-attachments/assets/cc727442-5544-4610-b0ee-748e0b20924a)

I think the inclusion of `| true` is more confusing than just doing `rm -rf` since the outcomes are the exact same.

```suggestion
  rm -rf themes
```